doozer/doozerlib/cli/images_streams.py

```diff
@@ -1346,6 +1346,10 @@ def check_if_upstream_image_exists(upstream_image):
                     pr_title = f'Bug {bug}: {pr_title}'
                 try:
                     new_pr = public_source_repo.create_pull(title=pr_title, body=pr_body, base=public_branch, head=fork_branch_head, draft=draft_prs)
```

Comment:
Are we also updating the PR here? It might be that PR is updated by a latter job, not a big issue though

Reply:
No this only happens at first PR creation